packages/paste-design-tokens/tokens/global/text-color.yml

```diff
@@ -16,7 +16,7 @@ props:
     value: "{!palette-gray-0}"
     comment: Inverse text color for dark backgrounds
   color-text-inverse-weak:
-    value: "{!palette-gray-60}"
+    value: "{!palette-gray-50}"
```

Comment:
Is the -60 value still fine on the Sendgrid theme?

Reply:
Yep, it passes AA (https://www.aremycoloursaccessible.com/?background=%2308152c&colorCombos=%5Bobject%20Object%5D&colorCombos=%5Bobject%20Object%5D&isLight=false&textColor=%239ca7ba)

Reply:
ok actually, just noticed `color-background-inverse` isn't defined in the SendGrid theme tokens. I'll add it.